src/popover/interfaces.ts

```diff
@@ -93,12 +93,12 @@ export interface Offset {
   top: number;
 }
 
-export interface BoundingBox {
+export interface Dimensions {
```

Comment:
Renamed these two interfaces hopefully for more clarity.

`BoundingOffset` was supposed to mean "bounding box with offset". However I would expect a bounding offset to already contain the offset (left and top coordinates) besides the width and the height.